src/AssertSaga.ts

```diff
@@ -113,16 +116,19 @@ export class SagaAssertImplementation {
         args:
             | SingleActionAssertionArguments
             | MultipleActionsAssertionArguments,
-        createEffect: (pattern: string[], handler: SagaType) => unknown
+        createEffect: (pattern: string[], handler: SagaType, ...handlerArgs: any[]) => unknown
     ) => {
         const actions = isSingleActionAssertion(args)
             ? [args.action]
             : args.actions;
+
         const actionTypes = actions
             .map(action => getType(action) as string)
             .sort();
 
-        const saga = this.saga();
+        const sagaArgs = args.handlerArgs === undefined ? [] : args.handlerArgs;
```

Comment:
We want to avoid verifying arguments when `handlerArgs` are not specified at all. Making them an empty array makes us verify them, doesn't it?

Reply:
If yes, please add a test for this case.